lib/validation/rules/rule.ts

```diff
@@ -1,8 +1,14 @@
 import { helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
 
-export function rule(validation: (value: any) => boolean, msg?: string) {
-  return helpers.withMessage(
-    () => msg ?? 'The value is invalid.',
-    (value: any) => !helpers.req(value) || validation(value)
-  )
+export const message = {
+  en: 'The value is invalid.',
+  ja: 'この値は正しくありません。'
+}
+
+export function rule(validation: (value: unknown) => boolean, msg?: string) {
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang],
+    validation: (value) => !helpers.req(value) || validation(value)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70